docs/reference/snapshot-restore/monitor-snapshot-restore.asciidoc

```diff
@@ -31,6 +31,8 @@ PUT /_snapshot/my_fs_backup
 }
 
 PUT /_snapshot/my_backup/snapshot_1?wait_for_completion=true
+
+PUT /_snapshot/my_backup/some_other_snapshot?wait_for_completion=true
```

Comment:
Since we moved back to 7.x error handling for single repo requests, this would fail now instead of return a `200` with errors in the response map. In fact this line is included in the 7.x version of this file (I assume for that very reason) and this nicely aligns the versions ago :)